cli/internal/runcache/runcache.go

```diff
@@ -303,10 +305,13 @@ func (tc TaskCache) SaveOutputs(ctx context.Context, logger hclog.Logger, termin
 // to this run and the given PackageTask
 func (rc *RunCache) TaskCache(pt *nodes.PackageTask, hash string) TaskCache {
 	logFileName := rc.repoRoot.Join(pt.RepoRelativeLogFile())
-	hashableOutputs := pt.HashableOutputs()
-	repoRelativeGlobs := make([]string, len(hashableOutputs))
-	for index, output := range hashableOutputs {
-		repoRelativeGlobs[index] = filepath.Join(pt.Pkg.Dir.ToStringDuringMigration(), output)
+	repoRelativeGlobs := pt.HashableOutputs()
+
+	for index, output := range repoRelativeGlobs.Inclusions {
+		repoRelativeGlobs.Inclusions[index] = filepath.Join(pt.Pkg.Dir.ToStringDuringMigration(), output)
```

Comment:
This makes me nervous. Doing a path translation in place means that the same type can have two different kinds of paths, which means we're probably going to be using the wrong path in the wrong place at some point. Is there a way to do this with a different type, even if it looks identical, so that we can understand which ones are intended to have which kinds of paths / globs?